website/docs/glossary.mdx

```diff
@@ -50,6 +50,22 @@ An **Access Control List**. For an overview of how they work in Solid, see
 
 A term with which we typically mean a person, but which could also refer to e.g. bots.
 
+#### Default access
+
+[Access](#access) rules that apply not to the [Container](#container) Resource directly, but to its
+children.
```

Comment:
Darn it, I'd typed a length reply to this but apparently didn't hit the submit button :( Let's try again...

Yes, I think this will be confusing - it's based on the spec nomenclature, so better suggestions are welcome.

So far, we've used the following terminology:

- Resource ACL: ACL that applies to a given Resource.
- Fallback ACL: ACL of the closest parent of a given Resource with an ACL, which will define what access rules apply to the Resource if it doesn't have its own ACL. In other words, the Fallback ACL of a Resource is also the Resource ACL of one of its parent Resources.
- Access Rule: a single grant of access in an ACL. So an ACL consists of multiple Access Rules.
- Resource Access Rule: an Access Rule that applies to the Resource that the containing ACL acts as the Resource ACL for.
- Default Access Rule: an Access Rule that applies to child Resources that the containing ACL acts as a Fallback ACL for.

Reply:
@Vinnl  - Would "Inheritable Access Rule" work for those that applies to child Resources?  As in you can specify Inheritable Access Rules for a parent container and all its child resources will default to/inherit that, unless they have their own resource-specific Access rule.

Reply:
That works well for documentation, but I suppose it's probably best to align the API with that terminology as well. It's already pretty unwieldy (`getAgentDefaultAccessOne`), so I'm not sure if `getAgentInheritableAccessOne` is that great. Although, typing it out, it doesn't look _that_ much longer...

@NSeydoux, what do you think?

Reply:
And what about `Inherited` instead of `Inheritable` ? It is (slightly) shorter, and I think it reads well (`getAgentInheritedAccessOne`).

I think overall you are right @kay-kim, having the notion of inheritance in the methods name will show intent more clearly than `default`. And given the length of our method names, I don't think 2 characters will make much difference ^^